src/Aspire.Hosting.Yarp/YarpResourceExtensions.cs

```diff
@@ -77,4 +77,16 @@ public static IResourceBuilder<YarpResource> WithHostPort(this IResourceBuilder<
             endpoint.Port = port;
         });
     }
+
+    /// <summary>
+    /// Enables static file serving in the YARP resource.
+    /// </summary>
+    /// <param name="builder">The resource builder for YARP.</param>
+    /// <returns>The <see cref="IResourceBuilder{T}"/>.</returns>
+    public static IResourceBuilder<YarpResource> WithStaticFiles(this IResourceBuilder<YarpResource> builder)
```

Comment:
Add an overload of `WithStaticFilesBindMount` that uses WithContainerFiles where the target folder is wwwroot.

Also update the comment to explain that the static file folder is wwwroot.